sui/typescript/backend_examples/src/invisible_wallet.ts

```diff
@@ -68,14 +66,10 @@ console.log("status:", txInfo.effects?.status.status);
 
 //  Use The Invisible Wallet API's method to do all the sponsor, sign, and execute 
 //   transaction steps with one call. Returns the associated transaction digest if successful.
-async function sponsorSignExecuteInOneRequest(signer: ShinamiWalletSigner, 
-                                                gaslessTx: GaslessTransaction) : Promise<string> {
+async function sponsorSignExecuteInOneRequest(signer: ShinamiWalletSigner,
+  gaslessTx: GaslessTransaction): Promise<string> {
   const sponsorSignAndExecuteResponse = await signer.executeGaslessTransaction(
     gaslessTx, // by not setting gaslessTx.gasBudget we take advantage of Shinami auto-budgeting
-    { showEffects: true },
```

Comment:
Did you remove this on purpose?

Reply:
Yes, because I don't use the effects. So, I'm fine with the default behavior and simplifying the example.